core/server/master/src/main/java/alluxio/master/metrics/DefaultMetricsMaster.java

```diff
@@ -126,65 +120,45 @@ public Object getValue() {
     }
   }
 
+  private void cleanUpOrphaneMetrics() {
+    mMetricsStore.cleanUpOrphanedMetrics();
+  }
+
   private void registerAggregators() {
     // worker metrics
-    addAggregator(new SumInstancesAggregator(MetricKey.CLUSTER_BYTES_READ_ALLUXIO.getName(),
-        MetricsSystem.InstanceType.WORKER, MetricKey.WORKER_BYTES_READ_ALLUXIO.getName()));
```

Comment:
All `BytesRead/Written` will become actual `MetricType.COUNTER`. Their values will be incremented when received worker/client reported metrics.